packages/dev/docs/src/docs.css

```diff
@@ -356,6 +370,57 @@ h2.sectionHeader {
   }
 }
 
+@define-mixin small-propTable {
```

Comment:
Mixin that is used to make the prop tables in popovers use the small size, just like on small screens.